lsp/syntaxes/civet.json

```diff
@@ -166,6 +166,24 @@
         }
       ]
     },
+
+    {
+      "name": "string.unquoted.module-reference.civet",
+      "match": "(?<=(^\\s*(import|export)|^)\\s*(type\\s+)?((\\w+\\s*,\\s*)?\\{(.|\\n)*\\}|\\w+|(\\*\\s+as\\s+\\w+))\\s+(from)\\s+)([^\\(\\{\\[\"';\\s=>][^\"';\\s=>]+|\".+?\"|'.+?')(?=[^()]|$)"
```

Comment:
`\\{(.|\\n)*\\}` seems rather inclusive. Perhaps `(.|\\n)` should be replaced by `[^{}]`? Or at least `*` should be `*?`, like the string regexes.

I also realize that, if you're just trying to match unquoted module references here, you don't actually need `|\".+?\"|'.+?'`. (If you keep them, shouldn't `+` be `*`?)